libs/hw-ledger-key-ring-protocol/package.json

```diff
@@ -24,7 +24,7 @@
     "bip32": "^4.0.0",
     "secp256k1": "^5.0.0",
     "create-hmac": "^1.1.7",
-    "tiny-secp256k1": "1"
+    "tiny-secp256k1": "1.1.7"
```

Comment:
Still I think it would be best to also update the `libs/ledgerjs/packages/hw-app-btc/package.json`. First to avoid duplicating the lib but also I'm struggling building LLD with newer versions of electron builder because of this package (for the app ASAR integrity check) so I would prefer having only one to worry about!